test/node_redis.spec.js

```diff
@@ -0,0 +1,675 @@
+                    client.once("connect", function () {
+                        client.flushdb(done)
+                    });
+                });
+
+                afterEach(function () {
+                    client.end();
+                });
+
+                describe("when redis closes unexpectedly", function () {
+                    it("reconnects and can retrieve the pre-existing data", function (done) {
+                        client.on("reconnecting", function on_recon(params) {
+                            client.on("connect", function on_connect() {
+                                async.parallel([function (cb) {
+                                    client.get("recon 1", function (err, res) {
+                                        helper.isString("one")(err, res);
+                                        cb();
+                                    });
+                                }, function (cb) {
+                                    client.get("recon 1", function (err, res) {
+                                        helper.isString("one")(err, res);
+                                        cb();
+                                    });
+                                }, function (cb) {
+                                    client.get("recon 2", function (err, res) {
+                                        helper.isString("two")(err, res);
+                                        cb();
+                                    });
+                                }, function (cb) {
+                                    client.get("recon 2", function (err, res) {
+                                        helper.isString("two")(err, res);
+                                        cb();
+                                    });
+                                }], function (err, results) {
+                                    client.removeListener("connect", on_connect);
+                                    client.removeListener("reconnecting", on_recon);
+                                    done(err);
+                                });
+                            });
+                        });
+
+                        client.set("recon 1", "one");
+                        client.set("recon 2", "two", function (err, res) {
+                            // Do not do this in normal programs. This is to simulate the server closing on us.
+                            // For orderly shutdown in normal programs, do client.quit()
+                            client.stream.destroy();
+                        });
+                    });
+
+                    describe("and it's subscribed to a channel", function () {
+                        // reconnect_select_db_after_pubsub
+                        // Does not pass.
+                        // "Connection in subscriber mode, only subscriber commands may be used"
+                        xit("reconnects, unsubscribes, and can retrieve the pre-existing data", function (done) {
```

Comment:
This test is one I ported from the old suite. IIRC, it was formatted incorrectly in the old suite and was reporting passing when it should have been reporting failing. Does the description of this test describe behavior that should work, such that we need to enter an issue to fix the bug in node_redis, or is node_redis behaving as it should, such that we should delete this test?